### DISCUSSION TITLE
How to cancel NetLicensing vendor account

### DISCUSSION THREAD

**Discussion by hbtj123** [Labs64/NetLicensing-Community#79] [Questions] [answered]
Dear Netlicensing, 

Please close the following account.
Are share how we can manage that as we are not using anymore.

thank you.

**Comment by r-brown** [comment, marked as answer]
To cancel NetLicensing vendor account please do following:
- request vendor account cancellation via email to info@netlicensing.io
- provide your vendor number and valid email ID
- we'll start account cancellation procedure and notify you about the proress

**Comment by hbtj123** [comment]
Hi Team, 

Sent the request from both of my email account.
Please let me know if i have missed anything.

Thanks!
